src/test/java/core/basesyntax/service/impl/BalanceOperationTest.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.FruitOperation;
+import org.junit.Test;
+
+public class BalanceOperationTest {
```

Comment:
Test for negative values

Reply:
that test in other test-cases (FruitRecordParserImplTest)